### PR TITLE
add doc for missing OMV core plugins

### PR DESCRIPTION
**Pull request by mi-hol**

### CODE REVIEW

**Review by flmaxey — APPROVED** [review]

File: plugins.rst [hunk 1]
```diff
@@ -22,11 +22,15 @@ Overview
...
 The following is the list of official plugins by |omv|.
 
-* **ClamAV**: Provides antivirus scan for folders.
+* **ClamAV**: Provides Clam AntiVirus (ClamAV). It is a free software, cross-platform and open-source antivirus software toolkit able to detect many types of malicious software, including viruses
+* **Diskstats**: Complementary plugin to extend system statistics collection by adding iostats graphs.
 * **Forked-daap**: Provides a daap protocol music server.
 * **LVM2**: LVM managing. Create volume groups and logical partitions.
-* **Diskstats**: Complementary plugin to extend current system statistics collection by adding iostats graphs.
 * **NUT**: Controlling and configuring UPS. The driver support is based on NUT.
+* **ShAirPort**: Provides Airtunes emulator. Stream music wirelessly to your iPod/iPad/iPhone/iTunes.
```
Comment (votdev) [inline plugins.rst:30]: I'm not sure, but i think it is named `Shairport`, at least according to the [Git repository](https://github.com/mikebrady/shairport-sync) of the project.

**Review by votdev — APPROVED** [review]
LGTM, except my comment.

**Review by votdev — APPROVED** [review]

File: plugins.rst [hunk 2]
```diff
@@ -22,11 +22,15 @@ Overview
 
 The following is the list of official plugins by |omv|.
 
-* **ClamAV**: Provides antivirus scan for folders.
+* **ClamAV**: Provides Clam AntiVirus (ClamAV). It is a free software, cross-platform and open-source antivirus software toolkit able to detect many types of malicious software, including viruses
+* **Diskstats**: Complementary plugin to extend system statistics collection by adding iostats graphs.
```
Comment (votdev) [inline plugins.rst:26]: Could you please change that to the following. I think this phrasing is more clear than the original (moved) text.
```suggestion
* **Diskstats**: Complementary plugin to extend system statistics collection by adding I/O statistic graphs.
```

File: plugins.rst [hunk 3]
```diff
@@ -22,11 +22,15 @@ Overview
...
 * **Forked-daap**: Provides a daap protocol music server.
 * **LVM2**: LVM managing. Create volume groups and logical partitions.
-* **Diskstats**: Complementary plugin to extend current system statistics collection by adding iostats graphs.
 * **NUT**: Controlling and configuring UPS. The driver support is based on NUT.
+* **Shairport**: Provides Airtunes emulator. Stream music wirelessly to your iPod/iPad/iPhone/iTunes.
+* **ShareRootFs**: Provides shared directires on root file sstem
```
Comment (votdev) [inline plugins.rst:31]:
```suggestion
* **ShareRootFs**: Provides shared directories on root file system.
```

File: plugins.rst [hunk 4]
```diff
@@ -22,11 +22,15 @@ Overview
...
+* **SNMP**: Provides Simple Network Management Protocol (SNMP). SNMP is an Internet Standard protocol for collecting and organizing information about managed devices on IP networks
```
Comment (votdev) [inline plugins.rst:32]: Most enumerations end with a dot, so we should do the same here.
```suggestion
* **SNMP**: Provides Simple Network Management Protocol (SNMP). SNMP is an Internet Standard protocol for collecting and organizing information about managed devices on IP networks.
```

File: plugins.rst [hunk 5]
```diff
@@ -22,11 +22,15 @@ Overview
...
+* **ClamAV**: Provides Clam AntiVirus (ClamAV). It is a free software, cross-platform and open-source antivirus software toolkit able to detect many types of malicious software, including viruses
+* **Diskstats**: Complementary plugin to extend system statistics collection by adding iostats graphs.
 * **Forked-daap**: Provides a daap protocol music server.
 * **LVM2**: LVM managing. Create volume groups and logical partitions.
-* **Diskstats**: Complementary plugin to extend current system statistics collection by adding iostats graphs.
 * **NUT**: Controlling and configuring UPS. The driver support is based on NUT.
+* **Shairport**: Provides Airtunes emulator. Stream music wirelessly to your iPod/iPad/iPhone/iTunes.
+* **ShareRootFs**: Provides shared directires on root file sstem
+* **SNMP**: Provides Simple Network Management Protocol (SNMP). SNMP is an Internet Standard protocol for collecting and organizing information about managed devices on IP networks 
+* **TFtp**: Provides Trivial File Transfer Protocol (TFTP). TFTP is a simple lockstep File Transfer Protocol which allows a client to get a file from or put a file onto a remote host
```
Comment (votdev) [inline plugins.rst:33]:
```suggestion
* **TFtp**: Provides Trivial File Transfer Protocol (TFTP). TFTP is a simple lockstep File Transfer Protocol which allows a client to get a file from or put a file onto a remote host.
```

**Review by votdev — CHANGES_REQUESTED** [review]

File: plugins.rst (hunk 3 above)
Comment (votdev) [reply plugins.rst:31]: This issue hasn't been fixed.

**Review by votdev — CHANGES_REQUESTED** [review]

**Comment by mi-hol** [comment]
weird, I had pressed "commit suggestion" as flagged by github in below screenshot, but the changes was actually not applied by github :(
I'd suggest to merge as the typo existed before I added line and to fix them in a second PR

image

**Comment by mi-hol** [comment]
after a page refresh, I was able to accept the change again, weird (as written)

**Pull request merged**